projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-cell.component.ts

```diff
@@ -27,10 +28,11 @@ export class IgxHierarchicalGridCellComponent extends IgxGridCellComponent imple
         public cdr: ChangeDetectorRef,
         private helement: ElementRef,
         protected zone: NgZone,
-        touchManager: HammerGesturesManager
+        touchManager: HammerGesturesManager,
+        protected platformUtil: PlatformUtil
         ) {
-            super(selectionService, crudService, gridAPI, selection, cdr, helement, zone, touchManager);
-            this.hSelection = <IgxHierarchicalSelectionAPIService>selection;
+            super(selectionService, crudService, gridAPI, selection, cdr, helement, zone, touchManager, platformUtil);
+            // this.hSelection = <IgxHierarchicalSelectionAPIService>selection;
```

Comment:
This doesn't seem related to the change

Reply:
Yech, the cherry pick from 8.3.x has introduced some hard to merge conflicts. I missed this one - thank you.